Main.java

```diff
@@ -32,55 +32,82 @@ public static void linie(int x) {
-                String removeID = input.nextLine();
-                MainMenu.removeClient(removeID);
-            }
-            if (Objects.equals(inp, "3")) {
-                space(2);
-                MainMenu. editClient();
-            }
-            if (Objects.equals(inp, "4")) {
-                space(2);
-                System.out.print(color.BLUE + "Type client's ID: " + color.RESET);
-                String findID = input.nextLine();
-
-                MainMenu.findClient(findID);
-                utilizatorMenu(findID);
-
-                MainMenu.foundClientData.clear();
-            }
-            if (Objects.equals(inp, "5")) {
-                space(2);
-                MainMenu.allClients();
+            switch (inp) {
+                case "1":
+                    space(2);
+                    Client.createClient();
+                    break;
+                case "2":
+                    System.out.print(color.BLUE + "Type client's ID to remove them: " + color.RESET);
+                    String removeID = input.nextLine();
+                    MainMenu.removeClient(removeID);
+                    break;
+                case "3":
+                    space(2);
+                    MainMenu. editClient();
+                    break;
+                case "4":
+                    space(2);
+                    System.out.print(color.BLUE + "Type client's ID: " + color.RESET);
+                    String findID = input.nextLine();
+                    MainMenu.findClient(findID);
+                    utilizatorMenu(findID);
+                    MainMenu.foundClientData.clear();
+                    break;
+                case "5":
+                    space(2);
+                    MainMenu.allClients();
+                    break;
             }
+//            if (Objects.equals(inp, "1")) {
+//                space(2);
+//                Client.createClient();
+//            }
+//            if (Objects.equals(inp, "2")) {
+//                space(2);
+//                System.out.print(color.BLUE + "Type client's ID to remove them: " + color.RESET);
+//                String removeID = input.nextLine();
+//                MainMenu.removeClient(removeID);
+//            }
+//            if (Objects.equals(inp, "3")) {
+//                space(2);
+//                MainMenu. editClient();
+//            }
+//            if (Objects.equals(inp, "4")) {
+//                space(2);
+//                System.out.print(color.BLUE + "Type client's ID: " + color.RESET);
+//                String findID = input.nextLine();
+//
+//                MainMenu.findClient(findID);
+//                utilizatorMenu(findID);
+//
+//                MainMenu.foundClientData.clear();
+//            }
+//            if (Objects.equals(inp, "5")) {
+//                space(2);
+//                MainMenu.allClients();
+//            }
         }
```

Comment:
pe viitor incearca sa nu dai push cu comentarii, este o pierdere de spatiu, in mod normal ai face un commit mai intai cu acest cod, daca vrei sa il salvezi, dupa care il stergi complet, si faci un commit nou cu codul de care ai nevoie, comentariile le lasi mai mult pentru explicatii